docker-compose.yml

```diff
@@ -1,16 +1,26 @@
-version: "3.9"
 services:
   frontend:
     container_name: ks-frontend
     build:
       context: ./frontend
+    command: ["npm", "run", "dev"]
```

Comment:
Does this not run it always? Overriding the serve command which We want to run when --Watch is not present?

Reply:
Yea, you are right. It would always be overridden in our dev-only docker-compose. There should be an option to turn it off.

I have added the docker functionality for `profiles`. I have created the containers for this feature to turn on and off. We can choose which one to run with the `--profile` parameter of the `docker compose`. I have updated README with this information.